web/pandas/_templates/layout.html

```diff
@@ -89,7 +89,7 @@
                 pandas is a fiscally sponsored project of <a href="https://numfocus.org">NumFOCUS.</a>
             </p>
         </footer>
-                
```

Comment:
can you split this and similar into a separate, easy-consensus PR

Reply:
Done, see #36588. The reason I committed here is because otherwise the PR will fail CI (because these files are now also being inspected); should I keep the commit here until the other PR gets merged?